src/Microsoft.Identity.Client/ApiEvent.cs

```diff
@@ -32,46 +32,56 @@ namespace Microsoft.Identity.Client
 {
     internal class ApiEvent : EventBase
     {
+        public const string ConstApiId = EventNamePrefix + "api_id";
+        public const string ConstAuthority = EventNamePrefix + "authority";
+        public const string ConstAuthorityType = EventNamePrefix + "authority_type";
+        public const string ConstUiBehavior = EventNamePrefix + "ui_behavior";
+        public const string ConstValidationStatus = EventNamePrefix + "validation_status";
+        public const string ConstTenantId = EventNamePrefix + "tenant_id";
+        public const string ConstUserId = EventNamePrefix + "user_id";
+        public const string ConstWasSuccessful = EventNamePrefix + "was_successful";
+
         public ApiEvent() : base(EventNamePrefix + "api_event") {}
 
         public int ApiId
         {
-            set => this["api_id"] = value.ToString();
+            set => this[ConstApiId] = value.ToString();
```

Comment:
That would not even pass the compilation, because we specifically define the acceptable value type, which does not allow `null`.